travis/run-large-python-tests.sh

```diff
@@ -21,7 +21,7 @@ pytest tests --large --ignore=tests/examples --ignore=tests/h2o --ignore=tests/k
   --ignore=tests/spark --ignore=tests/tensorflow --ignore=tests/azureml --ignore=tests/onnx \
   --ignore=tests/keras_autolog --ignore=tests/tensorflow_autolog --ignore=tests/gluon \
   --ignore=tests/gluon_autolog --ignore=tests/xgboost --ignore=tests/lightgbm \
-  --ignore tests/spark_autologging --ignore=tests/models
+  --ignore=tests/spacy --ignore=tests/spark_autologging --ignore=tests/models
```

Comment:
It seems to me like an "=" is missing here in the master's branch. Adding it made the Travis build pass. @smurching